tests/extend_test.py

```diff
@@ -82,5 +86,14 @@ def no_rule(*args, **kwargs):
     self.assertEqual(impl, jax.random.key_impl(k))
 
 
+class FfiTest(jtu.JaxTestCase):
+
+  @unittest.skipIf(xla_extension_version < 265, "Requires jaxlib 0.4.29")
+  def testHeadersExist(self):
+    base_dir = os.path.join(jex.ffi.include_dir(), "xla", "ffi", "api")
+    for header in ["c_api.h", "api.h", "ffi.h"]:
+      self.assertTrue(os.path.exists(os.path.join(base_dir, header)))
```

Comment:
If you want these files to exist when running this test in Bazel, you need to add the headers as a `data = [...]` dependency of `//jaxlib`.

Reply:
Thanks! That makes sense to me, although in detail I'm not sure how to actually make them end up in the right place. But AFAICT from looking at previous builds on `main` these tests (in [`tests/ffi_test.py`](https://github.com/google/jax/blob/main/tests/ffi_test.py)) were running (and passing) before this change, so I'm clearly confused about something!